src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -433,12 +440,11 @@ SELECT * FROM pg_dist_colocation
     ORDER BY colocationid;
  colocationid | shardcount | replicationfactor | distributioncolumntype | distributioncolumncollation
 ---------------------------------------------------------------------
-            3 |          4 |                 2 |                     23 |                           0
```

Comment:
Not sure where this was coming from. We never set `citus.shard_count` to 4 in multi_colocation_utils.